### PR TITLE
Win message and flow pipes

### PR DESCRIPTION
**Pull request by ilaypilo**

### CODE REVIEW

File: client_desktop/view/PipeDisplayer.java [hunk 1]
```diff
@@ -146,7 +134,7 @@ public void resize(double width, double height) {
 	}
 
 	public void cleanGame() {
-		flowPointList.clear();
+		//flowPointList.clear();
```
Comment (bzrihan-dn) [inline client_desktop/view/PipeDisplayer.java:137]: remove? or needed?

File: client_desktop/view/PipeDisplayer.java [hunk 2]
```diff
@@ -40,8 +38,7 @@
 	private Image pipeAngle180Image;
 	private Image pipeAngle270Image;
 
-	private Point startIndex;
-	private LinkedHashSet<Point> flowPointList = new LinkedHashSet<Point>();
+	//private LinkedHashSet<Point> flowPointList = new LinkedHashSet<Point>();
```
Comment (bzrihan-dn) [inline client_desktop/view/PipeDisplayer.java:41]: remove? or needed?

File: client_desktop/view/PipeDisplayer.java (hunk 1 above)
Comment (ilaypilo) [reply client_desktop/view/PipeDisplayer.java:137]: no need

File: client_desktop/view/PipeDisplayer.java (hunk 2 above)
Comment (ilaypilo) [reply client_desktop/view/PipeDisplayer.java:41]: no need

**Pull request merged**